### PR TITLE
Phoenix and Erlang/Elixir without sudo

### PR DESCRIPTION
**Pull request by waiteb3**
@hamiltont 
I would have done an install from the debs, but those assume you're installing to /usr/local, so this gets it done from source.

Also, because I promised Ariana.
```
+      o     +              o    
    +             o     +       +
o          +
    o  +           +        +
+        o     o       +        o
-_-_-_-_-_-_-_,------,      o 
_-_-_-_-_-_-_-|   /\_/\  
-_-_-_-_-_-_-~|__( ^ .^)  +     +  
_-_-_-_-_-_-_-" " "   "      
+      o         o   +       o
    +         +
o        o         o      o     +
    o           +
+      +     o        o      +    
```

### CODE REVIEW

**Comment by LadyMozzarella** [comment]
Can you rebase this please? I like just merged removal of the other Elixir framework and I think it caused a conflict.

**Comment by waiteb3** [comment]
@LadyMozzarella yeah, I should actually remember to push the rebase

File: frameworks/Erlang/elli/bash_profile.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```
Comment (LadyMozzarella) [inline frameworks/Erlang/elli/bash_profile.sh]: We're no longer using bash_profile.sh. exports need to be moved to setup.sh/install.sh

File: frameworks/Erlang/cowboy/bash_profile.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```
Comment (LadyMozzarella) [inline frameworks/Erlang/cowboy/bash_profile.sh]: This bash_profile also needs to be removed.

File: frameworks/Elixir/phoenix/bash_profile.sh [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```
Comment (LadyMozzarella) [inline frameworks/Elixir/phoenix/bash_profile.sh]: Remove this bash_profile also

**Comment by hamiltont** [comment]
Check out [this](https://github.com/hamiltont/FrameworkBenchmarks/blob/improve-java-setup/toolset/setup/linux/languages/java.sh) and [this](https://github.com/hamiltont/FrameworkBenchmarks/blob/improve-java-setup/frameworks/Java/activeweb/setup.sh#L4)

You may want to copy the idea for erlang - use the `erlang.installed` file to hold all the variables that need to be set in the various `setup.sh` files, such as PATH modifications, specific binary locations, etc

File: frameworks/Elixir/phoenix/setup.sh [hunk 4]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+export PATH=$PATH:$ERL_BIN:$ELX_BIN
+
+# sed -i 's|db_host: "localhost",|db_host: "${DBHOST}",|g' config/config.exs
```
Comment (LadyMozzarella) [inline frameworks/Elixir/phoenix/setup.sh]: Is this commented out for a reason?

**Pull request merged**